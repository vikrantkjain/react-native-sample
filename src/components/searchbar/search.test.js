import {fireEvent, render, screen} from '@testing-library/react-native';
import Searchbar from './index';

describe('Test SearchBar', () => {
  const onChangeTestMock = jest.fn();
  const CHANGE_TEXT = 'content';
  const tree = render(<Searchbar onChangeText={onChangeTestMock} />);
  it('Render successfully', () => {
    expect(tree).toBeDefined();
  });
  it('Render correctly', () => {
    const treeAsJSON = tree.toJSON();
    expect(treeAsJSON).toMatchSnapshot();
  });
  fireEvent.changeText(screen.getByTestId('TEST-SEARCHBAR'), CHANGE_TEXT);
});
