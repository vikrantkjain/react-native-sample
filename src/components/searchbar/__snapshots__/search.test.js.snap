// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SearchBar Render correctly 1`] = `null`;
